### PR TITLE
[REF] 8.0 mgmtsystem audit

### PR DESCRIPTION
**Pull request by llacroix**
Removed vim lines. And added a name for the act_window. Name can't be null.

It depends on this pull request: 
- [x] #25
- [x] #35
- [x] #26
- [x] OCA/knowledge#30

### CODE REVIEW

**Comment by yvaucher** [comment]
:+1:  waiting on dependencies

**Comment by max3903** [comment]
Let's fix #36 first and then include the fix in here.

**Comment by ghost** [comment]
Hi!

Is there any pending work in this branch?

**Comment by max3903** [comment]
Just waiting for #25, I think.

**Comment by ghost** [comment]
> Just waiting for #25, I think.

Hi @max3903! 

I made a pull request (savoirfairelinux/management-system#19) to include spanish translation.

File: mgmtsystem_audit/report/__init__.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
 # -*- encoding: utf-8 -*-
```
Comment (dreispt) [inline mgmtsystem_audit/report/__init__.py]: coding

File: mgmtsystem_audit/__openerp__.py [hunk 2]
```diff
@@ -43,7 +43,5 @@
     "demo": [
         'demo_audit.xml',
     ],
-    'installable': False,
-    "certificate": ''
+    'installable': True,
```
Comment (dreispt) [inline mgmtsystem_audit/__openerp__.py:42]: Move description to a README

File: mgmtsystem_audit/report/audit_report.py [hunk 3]
```diff
@@ -38,4 +38,3 @@ def __init__(self, cr, uid, name, context):
     parser=mgmtsystem_audit_report
 )
```
Comment (bwrsandman) [inline mgmtsystem_audit/report/audit_report.py]: Extra blank line here needs to be removed

File: mgmtsystem_audit/report/verification_list.py [hunk 4]
```diff
@@ -72,4 +78,3 @@ def get_lines_by_procedure(self, verification_lines):
     parser=mgmtsystem_audit_verification_list
 )
```
Comment (bwrsandman) [inline mgmtsystem_audit/report/verification_list.py]: Extra blank line here needs to be removed

File: mgmtsystem_audit/README.rst [hunk 5]
```diff
@@ -0,0 +1,76 @@
+mage:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (dreispt) [inline mgmtsystem_audit/README.rst]: image (missing "i")

**Comment by dreispt** [comment]
:+1: other than one small comment in the README

**Comment by max3903** [comment]
:+1:

**Pull request merged**